core/ui/card_ui/card_ui.gd

```diff
@@ -77,6 +78,10 @@ func _ready() -> void:
 
 	get_viewport().gui_focus_changed.connect(_on_focus_changed)
 	library_manager.reload_library()
+	
+	# Remove unneeded/conflicting elements from default menues
+	var settings_remove_list: PackedStringArray = ["DisksButton"]
```

Comment:
Why would we want to remove this in the full session?

Reply:
I assumed we would do something completely different in the full session. SRM is pretty limited on how it interacts due to steam requirements.